homeassistant/components/mopeka/sensor.py

```diff
@@ -0,0 +1,140 @@
+"""Support for Mopeka sensors."""
+from __future__ import annotations
+
+from mopeka_iot_ble import SensorUpdate
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothDataProcessor,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothProcessorCoordinator,
+    PassiveBluetoothProcessorEntity,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    UnitOfLength,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.sensor import sensor_device_info_to_hass_device_info
+
+from .const import DOMAIN
+from .device import device_key_to_bluetooth_entity_key
+
+SENSOR_DESCRIPTIONS = {
+    "battery": SensorEntityDescription(
+        key="battery",
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    "battery_voltage": SensorEntityDescription(
+        key="battery_voltage",
+        device_class=SensorDeviceClass.VOLTAGE,
+        native_unit_of_measurement=PERCENTAGE,
```

Comment:
Percentage is not an allowed unit for voltage device class. I'm assuming it's just copy paste leftover from above.

Reply:
Yup Its a paste error. Will adjust shortly

Reply:
https://github.com/home-assistant/core/pull/86663